### DISCUSSION TITLE
How to get more Star in repository.

### DISCUSSION THREAD

**Discussion by Preetiraj3697** [Schweinepriester/github-profile-achievements#483] [Q&A]
leave a star on my repo and reply with your repo and I will star it back https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Comment by 1RB** [comment]
Star mine and I'll star yours
* https://github.com/1RB/Authworria

*reply afterwards so I get a notification when you have done so*

**Reply by Preetiraj3697** [reply]
I gave you a star, now it's your turn

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by abidurrahman11** [comment]
You gave someone a star and he gave you one back.
Guys it's not what you want, you know right?
Make some great repo that will attract others, optimize your readme file. You'll surely get stars.
Best of Luck😍

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.
